package/base-files/files/lib/preinit/30_failsafe_wait

```diff
@@ -32,8 +32,8 @@ fs_wait_for_key () {
 		while [ $timer -gt 0 ]; do
 			pi_failsafe_net_message=true \
 				preinit_net_echo "Please press button now to enter failsafe"
-			echo "$timer" >$keypress_sec
-			timer=$(($timer - 1))
+			echo "$timer" > $keypress_sec
+			timer=$((timer - 1))
```

Comment:
nice. I assume the automatic $ removal in (()) is automatic?

Reply:
Correct